modules/commons/src/main/resources/schemas/neurosciencegraph/commons/variables/v0.2.0.json

```diff
@@ -0,0 +1,136 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/variables/v0.2.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "AlgebraicValueShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "property": [
+        {
+          "path": "schema:unitCode",
+          "name": "Unit code",
+          "datatype": "xsd:string",
+          "minCount": 1,
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:statistic",
+          "name": "Statistic",
+          "description": "The name of the statistic describing the sample.",
+          "datatype": "xsd:string",
+          "maxCount": 1
+        }
+      ]
+    },
+    {
+      "@id": "ValueShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "AlgebraicValueShape"
+        },
+        {
+          "property": [
+            {
+              "path": "schema:value",
+              "name": "Value",
+              "datatype": "xsd:float",
+              "minCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "VariableShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "property": [
+        {
+          "path": "nsg:quantityType",
+          "name": "Quantity type",
+          "description": "The type of the measured quantity represented by the variable.",
+          "minCount": 1,
+          "maxCount": 1
```

Comment:
Longer term, this shape should probably re-use the TypedLabeledOntologyTerm schema (https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.4.json) with an included OntologyTermShape for the quantity types

Reply:
Looks good. To be seen with @MFSY for the specific XyzOntologyTermShape.